ui/components/NotificationCenter/formatters/model_registration.js

```diff
@@ -193,7 +193,7 @@ export const ModelImportedSection = ({ modelDetails }) => {
         return (
           <Box key={index} mb={2}>
             <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'space-between' }}>
-              <Typography gutterBottom>
+              <Typography gutterBottom data-testid="model-info">
```

Comment:
CUrious if we can name this after the component as this may be to generic here. Also raise it one level vs just having the label included.